src/inset_state/inset_state.cpp

```diff
@@ -378,7 +456,7 @@ void InsetState::remove_tiny_polygons(const double &minimum_polygon_size)
     // Iterate over Polygon_with_holes
```

Comment:
It is unclear to me how this function takes advantage of the sorting of polygons on line 453. Do we require sorting?

Reply:
It is easier to access the private member than calling the getter function. I feel, therefore, it might be faster to call the private member directly.

```cpp
// Sort polygons with holes according to area
gd.sort_pwh_descending_by_area();
const auto &pwhs = gd.polygons_with_holes();

// Iterate over Polygon_with_holes
for (unsigned int i = 0; i < pwhs.size(); ++i) {
  if (i == 0 || pwh_area(pwhs[i]) > threshold) {
    gd_cleaned.push_back_polygon_with_holes(pwhs[i]);
  }
}

```
The sorting is done to ensure that the 0th indexed pwh is the largest pwh. Then, we can make sure that at least one of the pwh remains in the geodiv, in the case that all the pwh areas are below threshold.